src/ts/waterfall/sub-components/svg-marks.ts

```diff
@@ -1,8 +1,8 @@
-import {roundNumber} from "../../helpers/misc";
```

Comment:
Will revert then. I suppose these edge-cases are always the tough ones when working with multiple editors or IDEs. 

Created #122

The only one I could find on the fly is this, but it does not check in the curly brackets (still found 2 odd formating) - https://palantir.github.io/tslint/rules/import-spacing/.